webapp/src/ts/services/enketo.service.ts

```diff
@@ -135,25 +135,6 @@ export class EnketoService {
           $element.text(this.translateService.instant('enketo.' + $element.attr('data-i18n')));
         });
 
-        // TODO remove this when our enketo-core dependency is updated as the latest
-        //      version uses the language passed to the constructor
-        const languages = $html.find('#form-languages option');
-        if (languages.length > 1) { // TODO how do we detect a non-localized form?
-          // for localized forms, change language to user's language
-          $html
-            .find('[lang]')
-            .removeClass('active')
-            .filter('[lang="' + language + '"], [lang=""]')
-            .filter((idx, element) => {
-              // localized forms can support a short and long version for labels
-              // Enketo takes this into account when switching languages
-              // https://opendatakit.github.io/xforms-spec/#languages
-              return !$(element).hasClass('or-form-short') ||
-                ($(element).hasClass('or-form-short') && $(element).siblings( '.or-form-long' ).length === 0 );
-            })
-            .addClass( 'active' );
-        }
-
         const hasContactSummary = $(model).find('> instance[id="contact-summary"]').length === 1;
```

Comment:
I think we actually have to either: 
1. Figure out why Enketo calling this code doesn't work and make it work somehow. 
2. Include the whole contents of the `setAll` function here, so we cover all cases. 
3. A mix of 1 and 2